graphql_api/types/owner/owner.py

```diff
@@ -274,6 +274,19 @@ def resolve_owner_account(owner: Owner, info) -> dict:
     return Account.objects.filter(pk=account_id).first()
 
 
+@owner_bindable.field("isUserOktaAuthenticated")
+@sync_to_async
+@require_part_of_org
+def resolve_is_user_okta_authenticated(owner: Owner, info) -> bool:
+    okta_signed_in_accounts = info.context["request"].session.get(
+        "okta_signed_in_accounts"
+    )
+    if not okta_signed_in_accounts:
+        return False
+
+    return owner.account_id in okta_signed_in_accounts
```

Comment:
`owner.account_id` can potentially be `None`, so we'll probably want a check on that.

If we return default, then we can simplify this a bit:
```suggestion
    okta_signed_in_accounts = info.context["request"].session.get(
        "okta_signed_in_accounts", [],
    )
    if owner.account_id:
        return owner.account_id in okta_signed_in_accounts
    return False
```

Reply:
checking `if None in []` should also return False, unless we can have None in account ids session?

Reply:
We _shouldn't_ have `None` in the list, but it's likely one of the ways that this breaks (i.e. we accidentally start allowing None into the list or session corruption when loading from DB). I might be potentially overly conservative with this check, so I don't mind leaving this out.